scripts/settle.py

```diff
@@ -116,7 +116,7 @@ def update_market_oracle(market_address, oracle):
             bet = market.functions.getBetByIndex(i).call()
 
             # check if bet is less than 2 hours old
-            if bet[2] > now - 60 * 60 * 24 and bet[3] == False:
+            if bet[2] > now - 60 * 60 * 72 and bet[3] == False:
```

Comment:
does this check if a bet is less than 2 hours old?

Reply:
Looks like 3 days...

Reply:
Should have pushed that.  Fixed